src/session.c

```diff
@@ -388,12 +402,13 @@ static CK_RV mutex_destroy(P11PROV_CTX *provctx, pthread_mutex_t *lock,
     if (err != 0) {
         err = errno;
         ret = CKR_CANT_LOCK;
-        P11PROV_raise(provctx, ret, "Failed to destroy %s lock (errno=%d)", obj,
-                      err);
+        MUTEX_RAISE_ERROR("Failed to destroy");
     }
     return ret;
 }
-#define MUTEX_DESTROY(obj) mutex_destroy((obj)->provctx, &(obj)->lock, #obj)
+#define MUTEX_DESTROY(obj) \
+    mutex_destroy((obj)->provctx, &(obj)->lock, #obj, OPENSSL_FILE, \
+                  OPENSSL_LINE, OPENSSL_FUNC)
 
 /* Session stuff */
 #define DEFLT_SESSION_FLAGS CKF_SERIAL_SESSION
```

Comment:
should this say `DEFAULT_SESSION_FLAGS` or the abbreviation is intentional or more readable than expanding?

Reply:
I think I wanted to keep things shorter when I originally introduced it.
A bit silly, I know, but I would change it in another PR if it is an issue.